.circleci/config.yml

```diff
@@ -1,8 +1,12 @@
 version: 2.1
 
 references:
+  repo_path: &repo_path
+      /home/circleci/repo
+  ng_workspace_path: &ng_workspace_path
+      /home/circleci/repo/ddp-workspace
```

Comment:
Setting these up as " constants" is what we are doing in the ddp-study-server. Can be referenced wherever needed.